tic.R

```diff
@@ -1,5 +1,5 @@
 do_package_checks(error_on = "warning")
 
 if (ci_has_env("BUILD_PKGDOWN")) {
-  do_pkgdown(orphan = TRUE)
+  do_pkgdown(orphan = TRUE, install = TRUE)
```

Comment:
We shouldn't change this everywhere now since I am about to fix this upstream: https://github.com/ropenscilabs/tic/pull/185

The default should work then again.